src/session/src/context.rs

```diff
@@ -92,6 +92,12 @@ impl QueryContext {
             )
         }
     }
+
+    pub fn get_db_string(&self) -> String {
+        let catalog = self.current_catalog();
+        let schema = self.current_schema();
+        format!("{catalog}-{schema}")
```

Comment:
IIRC some places are using `{catalog}.{schema}` as the identifier, will this difference cause some problems?

Reply:
We are use `-` connected string on cloud so this would be exact how users see dbname on cloud console. But I'm going to correct it to `{schema}` when catalog is default so it would be fully consistent for both open source and cloud user.